### ISSUE TITLE
Main Status Entity - Clarification

### ISSUE THREAD

**Issue by Jiriteach** [vwt12eh8/hassio-ecoflow#20]
Hi - Great work on developing this add-on. Works really well.
I have 2 x Ecoflow River's and stats are coming through nice.

I do have a question though on the main status entity.

Both of my Ecoflow's are showing as "Problem" but there is nothing on the units themselves or the official Ecoflow app to indicate any problem.

Could there be a an issue with this entity?

Thank

<img width="512" alt="Screen Shot 2022-07-29 at 18 42 18" src="https://user-images.githubusercontent.com/2116446/181699388-3d8e73e6-af92-4d45-b73f-7db81db35e4c.png">
s

**Comment by Jiriteach** [comment]
Looking at diagnostics - I am seeing a battery_main_error of 6 under EMS. 
What does this mean?

`"ems": {
      "battery_main_error": 6,
      "battery_main_version": "1.0.2.68",
      "battery_main_level": 100,
      "battery_main_voltage": 33.12,
      "battery_main_current": 4294967271,
      "battery_main_temp": 31,
      "_open_bms_idx": 1,
      "battery_capacity_remain": 9589,
      "battery_capacity_full": 9450,
      "battery_cycles": 2,
      "battery_level_max": 80,
      "battery_main_voltage_max": 4.128,
      "battery_main_voltage_min": 4.112,
      "battery_main_temp_max": 31,
      "battery_main_temp_min": 28,
      "mos_temp_max": 32,
      "mos_temp_min": 32,
      "battery_main_fault": 0,
      "_bq_sys_stat_reg": 128,
      "_tag_chg_amp": 15000
    }`

**Comment by Jiriteach** [comment]
I think I have found the issue even though I dont yet know what a battery_main_error of 6 under EMS means.

I am running my Ecoflow River's as UPS so AC connected to mains and AC out to power devices. Once the battery is charged, the Ecoflow River's run in a bypass mode.

I was able to remove the "Problem" status by failing over the mains so effectively things are running on battery. Restore mains and it charges the battery back up until it hits 100% and then the status is "Problem" again. It could be the way the API is reading things as at 100% with the mains back on and the battery charged - its just bypassing power so shouldnt be showing as "Problem".

Hope this helps resolve the issue.
